graphics/MoltenVK/Portfile

```diff
@@ -65,15 +55,49 @@ destroot {
     }
 }
 
-platform darwin {
-    if {${os.major} < 16} {
-        archive_sites
-        distfiles
-        depends_build
-        known_fail  yes
-        pre-fetch {
-            ui_error "${subport} @${version} requires macOS Sierra or later"
-            return -code error "incompatible OS X version"
+# GitHub artifacts deployment target is 10.13
+if {${os.major} < 17} {
+
+    github.setup        KhronosGroup MoltenVK 1.1.9 v
+    epoch               1
+    revision            1
+
+    set sdkversion      1.3.211.0
+
+    distname            vulkansdk-macos-${sdkversion}
+    use_dmg             yes
+    use_tar             no
+
+    # url only works for the latest avalible SDK, older versions will 404
+    master_sites        https://sdk.lunarg.com/sdk/download/${sdkversion}/mac/
+
+    checksums           sha256  bfe654af00030b6e65521f834f0830f15e18c828594226865f15c92a9ea68363 \
+                        rmd160  3674f67b37f7bcc1746d55c3baa975d0cca9dbaa \
+                        size    275553243
+
+    depends_build       port:p7zip
+    depends_skip_archcheck p7zip
+
+    variant universal   {}
+    use_configure       no
+
+    build {
+        # bypass the installer that requires macOS 10.13
+        system "${prefix}/bin/7z x -aoa ${worksrcpath}/InstallVulkan.app/Contents/Resources/installer.dat -o${workpath}/VulkanSDK"
+    }
+
+    destroot {
+        set output_dir ${workpath}/VulkanSDK
+
+        file copy ${output_dir}/macOS/bin/MoltenVKShaderConverter ${destroot}${prefix}/bin
+        file attributes ${destroot}${prefix}/bin/MoltenVKShaderConverter -permissions +x
+        file copy ${output_dir}/macOS/lib/libMoltenVK.dylib ${destroot}${prefix}/lib
+
+        # vulkan and vk_video are provided via vulkan-headers
+        file copy ${output_dir}/MoltenVK/include/MoltenVK ${destroot}${prefix}/include
+
+        if {![variant_isset universal]} {
```

Comment:
Needs to also check `[variant_exists universal]`, since the universal variant will be removed when using an SDK that only supports a single arch. Same on line 53 (which can also have its parens removed).

Reply:
I’ll fix this while splitting into stub/subports

Reply:
That change doesn’t really apply to the way this port works.

Reply:
I don't see why not. It decides whether to install universal files based on the universal variant. That this is done by either thinning the binary or not doesn't make any difference to the final result.

Come to think of it though, the archs provided by the binary don't necessarily match the ones in `configure.universal_archs`, so more effort is needed to make sure the archs of the installed files match those recorded in the registry for the port.

Reply:
I’d added a “universal” variant manually, removed that now and did the changes you’ve mentioned.

> Come to think of it though, the archs provided by the binary don't necessarily match the ones in `configure.universal_archs`, so more effort is needed to make sure the archs of the installed files match those recorded in the registry for the port.

I’m having it run thin against `configure.build_arch` when the system supports arm64 & x86_64 that steps skipped shouldn’t that be enough?

Reply:
When it supports both arm64 & x86_64 and +universal is set the thin stage is skipped

Reply:
> I’d added a “universal” variant manually, removed that now and did the changes you’ve mentioned.

Whether the universal variant is added manually or not makes no difference, it is removed if `configure.universal_archs` doesn't end up containing 2 or more archs. That will be the case on systems like 10.14 where universal building isn't possible, or if `universal_archs` is set to an empty value in macports.conf.

Reply:
> I’m having it run thin against `configure.build_arch` when the system supports arm64 & x86_64 that steps skipped shouldn’t that be enough?

OK, so yes, the `supported_archs` will ensure that the universal variant is removed when unsupported universal archs are configured (like on 10.13 where the default is `i386 x86_64`.) But a universal variant never exists now, because you removed it. If you put it back, all should be well.